source/actions3/actions/AMScanAction.h

```diff
@@ -82,46 +135,129 @@ protected slots:
+	  * scan is known to the database, then an attempt to resave it is made. Adapts
+	  * the cancelled instruction to the Action3 interface.
+	  */
 	void onControllerCancelled();
-	/// Handles at the action level if the controller fails.
+
+	/*!
+	  * Handles the controller signalling that the scan has failed. If the scan
+	  * is known to the database, then an attempt to resave it is made. Adapts the
+	  * failed instruction to the Action3 interface.
+	  */
 	void onControllerFailed();
-	/// Handles all the wrap up and clean up if the controller succeeds.
+
+	/*!
+	  * Handles the controller signalling that the scan has succeeded. Saves the
+	  * finshed scan to the database and, if the configuration specifies an auto-export,
+	  * then an export is performed. Adapts the succeeded instruction to the Aciton3
+	  * interface.
+	  */
 	void onControllerSucceeded();
-	/// Handles setting the status text when the initialization actions are started.
+
+	/*!
+	  * Handles the controller signalling that it has begun the initialization
+	  * steps. Sets the status text for the action to "Initializing".
+	  */
 	void onControllerInitializing();
-	/// Handles setting the status text when cleaning actions are started.
+
+	/*!
+	  * Handles the controller signalling that it has begun the clean-up steps.
+	  * Sets the status text for the aciton to "Cleaning up".
+	  */
 	void onControllerCleaningUp();
-	/// Handles the progress changed passed up from the controller.
+
+	/*!
+	  * Handles the controller signaling a periodic status changed update. Re-emits
+	  * the progress changed signal on the Action3 interface.
+	  */
 	void onControllerProgressChanged(double elapsed, double total);
-	/// Helper slot that updates the status text when the controller changes state.
+
+	/*!
+	  * Handles the controller signaling that it has made a state transition.
+	  * Updates the status text to reflect the new state.
+	  *
+	  * /// Is this still the case. There's nothing in the body of this method ///
+	  */
 	void onControllerStateChanged();
 
-	/// Handles making sure that we're ready for deletion
+	/*!
+	  * Prepares the scan action for deletion. The scan action will be added to
+	  * the deletion queue when the logging for this scan has been completed.
+	  * // I'm going off the body here. The name seems to suggest some other logic///
+	  */
 	void checkReadyForDeletion();
+
 //	void onReadyForDeletionChanged(bool isReady);
 
 protected:
-	/// This function is called from the Starting state when the implementation should initiate the action. Once the action is started, you should call notifyStarted().
+
+	/// A lot of these comments (which come down from AMAction3 and are the same
+	/// across most AMAction sub-classes) all mention that you should call notifyX()
+	/// for most of these implemetation, but none of these notify functions exist.
+	/// Is this a remenant from an earlier time?
+
```

Comment:
I altered these comments to remove the instruction to call notifyX(), but an example can be seen in AMAxisStartedAction::startImplementation() [here](https://github.com/acquaman/acquaman/blob/master/source/actions3/actions/AMAxisStartedAction.h#L53).